neuromation/api/core.py

```diff
@@ -112,6 +116,11 @@ def _auth_headers(self) -> Dict[str, str]:
         else:
             real_headers = CIMultiDict()
         real_headers.update(self._headers)
+        trace_request_ctx = SimpleNamespace()
+        trace_id = self._trace_id
+        if trace_id is None:
+            trace_id = gen_trace_id()
+        trace_request_ctx.trace_id = trace_id
```

Comment:
Why you generate unique-per-request trace_id here?

Reply:
What do you propose?